packages/bot-web-ui/tsconfig.json

```diff
@@ -2,14 +2,14 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "outDir": "./dist",
-        "rootDir": "./src",
         "baseUrl": "./",
         "paths": {
             "Components/*": ["src/components/*"],
             "Constants/*": ["src/constants/*"],
             "Stores/*": ["src/stores/*"],
```

Comment:
@jim-deriv Seems like we also need to add a direct path too, Like `"Stores": ["src/stores"]` :(

<img width="775" alt="image" src="https://user-images.githubusercontent.com/72082844/187852458-7288d9e7-e76b-491c-947c-dffa3c46b9dc.png">

Reply:
@farzin-fs,

When do you get this error?

When building?

Reply:
@jim-deriv I just changed a component to `tsx` to test the config.

Reply:
<img width="1238" alt="app" src="https://user-images.githubusercontent.com/104334373/187860449-254b265b-c2bb-4670-b5a1-8a627049579e.png">
is it app.jsx?

Reply:
@jim-deriv Yes, I changed it to `app.tsx`.

for `bot-skeleton` and `smartcharts` I couldn’t find a solution

Reply:
Yeah. I'm looking into those

Reply:
@farzin-fs, The issue with bot-skeleton was because I had changed app.js to app.ts, but had not updated package.json